app/views/layouts/application.html.erb

```diff
@@ -96,6 +96,7 @@
         <%= render "shared/flashes" %>
         <%= render "shared/terms_of_deposit" %>
         <%= render "shared/help_modal" %>
+        <a name="content"></a>
```

Comment:
@peetucket Can you say more about why we need this anchor? I thought the skip-to-content links were working fine whenever the DOM contained `<main id="content">`. Is that not the case? Or do we have pages that lack this in the DOM?

Reply:
It's not clear that we do, but it was listed as a ding (that the "skip to" link doesn't lead to a specific location, so I added it as a precaution in the layout in case there is a page somewhere that doesn't have a `<main id="content">`

i don't think it will cause any harm if duplicated

Reply:
Yeah, I guess this is why I was so concerned about having tools for auditing this kind of stuff, so we don't add cruft where not needed.

Reply:
> It's not clear that we do, but it was listed as a ding (that the "skip to" link doesn't lead to a specific location, so I added it as a precaution in the layout in case there is a page somewhere that doesn't have a `<main id="content">`
> 
> i don't think it will cause any harm if duplicated

FWIW, in my earlier a11y PRs in H2, I found one `<main>` element lacking the `id="content"` and addressed it. So this may already be addressed without needing the anchor.